### PR TITLE
I want to update the Getting started with first workload doc

### LINKED ISSUE #1037: I want to update the Getting started with first workload doc to use Azure Voting App and DevOps Repo
**As a:**
Engineer
**I want:**
To update the documentation for deploying first workload 
**So that:**
We are consistently using the same application as we walk the users through various Bedrock scenarios
**Describe the solution you'd like:**
Update the document to use Azure Voting App manifest
https://github.com/microsoft/bedrock/tree/docs_spk/docs/firstWorkload
**Acceptance Criteria:**

- [x] The instructions are clear and new users should be able to finish the walthrough

- [x] instructions are modified use Azure Dev Ops repo

- [x] manifest file points to Voting App

**Describe alternatives you've considered:**
None
**Additional context:**
Current walkthrough uses Hello World which we do not use again in the subsequent walkthroughs
**Does this require updates to documentation?:**
Yes

### PR DESCRIPTION
**Pull request by edaena**
Closes #1037

### CODE REVIEW

File: docs/firstWorkload/README.md [hunk 1]
```diff
@@ -94,14 +96,18 @@ $ pbcopy < ~/cluster-deployment/keys/gitops-ssh-key.pub
 $ cat ~/cluster-deployment/keys/gitops-ssh-key.pub | xclip
 ```
 
-2. On the Github repository, select `Settings` -> `Deploy Keys` -> `Add deploy key`. Set the title and paste the contents of the public key. Check the  **Allow Write Access** option.
+1. Next, on AzureDevOps repository, open your security settings by browsing to the web portal and selecting your avatar in the upper right of the user interface. Select Security in the menu that appears.
```
Comment (timfpark) [inline docs/firstWorkload/README.md]: typo: selecting -> select

File: docs/firstWorkload/README.md [hunk 2]
```diff
@@ -643,9 +687,8 @@ kube-system   metrics-server         ClusterIP      10.0.189.185   <none>
 
 External load balancers like this take time to provision. If the EXTERNAL-IP of service is still pending, keep trying periodically until it is provisioned.
 
-The EXTERNAL-IP, in the case above, is: 52.175.216.214. By appending the port our service is hosted on we can use http://52.175.216.214:8080 to fetch the service in a browser.
+The EXTERNAL-IP, in the case above, is: 52.143.80.54. By appending the port our service is hosted on we can use http://52.143.80.54:80 to fetch the service in a browser.
 
-![Deployed Web application running](./images/WebAppRunning.png)
```
Comment (jred55) [inline docs/firstWorkload/README.md]: Can we pls upload the image of webapp one would see here for the voting app

File: docs/firstWorkload/README.md [hunk 3]
```diff
@@ -555,54 +561,92 @@ ts=2019-06-18T06:33:18.819404372Z caller=images.go:18 component=sync-loop msg="p
...
+  type: LoadBalancer
+  ports:
+  - port: 80
+  selector:
+    app: azure-vote-front
 ---
 ```
 
-This defines a 3 pod deployment of a web service with the container `andrebriggs/goserver:v1.2` with a LoadBalanced service.  Commit this file and push it to our remote GitOps repo:
+This defines a multi-container application that includes a web front end and a Redis instance is run in the cluster.
```
Comment (jred55) [inline docs/firstWorkload/README.md]: typo - change to "Redis instance running in the cluster"

File: docs/firstWorkload/README.md [hunk 4]
```diff
@@ -613,27 +657,27 @@ Watch the Flux pod logs again, but this time tailing them so we get updates with
 $ kubectl logs flux-5897d4679b-tckth -n flux -f
 ```
 
-Once Flux starts its next reconcilation, we see at the end of the output that Flux found the repo `bedrock-deploy-demo` and created the new service:
+Once Flux starts its next reconcilation, we should see at the end of the output that Flux has found the repo `app-cluster-manifests` and created the new service:
 
 `"kubectl apply -f -" took=1.263687361s err=null output="service/mywebapp created\ndeployment.extensions/mywebapp-v1 created"`.
```
Comment (jred55) [inline docs/firstWorkload/README.md]: change reference to mywebapp-1 here

**Pull request merged**